frontend/src/views/Administration.vue

```diff
@@ -238,9 +466,19 @@ export default {
       this.detailedMessage = undefined
       this.edit = false
     }
-  },
-  mounted () {
-    this.floatingButton = true
   }
 }
 </script>
+
+<style lang="scss" scoped>
+  .markdown {
```

Comment:
this is currently unused

Reply:
it is use in SLA card
```html
<v-list-item-title class="markdown wrap-text" v-html="slaDescriptionCompiledMarkdown"/>
```

Reply:
a sorry you are right, made a mistake while searching